packages/store/src/workspace/page.ts

```diff
@@ -461,28 +492,44 @@ export class Page extends Space<PageData> {
     });
   }
 
-  addSiblingBlock(
+  addSiblingBlocks(
     targetModel: BaseBlockModel,
-    blockProps: Partial<BaseBlockModel>,
+    props: Partial<BaseBlockModel> | Array<Partial<BaseBlockModel>>,
```

Comment:
The type here should not be mixed IMO.

Reply:
updated at https://github.com/toeverything/blocksuite/pull/1188/commits/61967baa696e1136b59eea31dcf7bea499b3a54f